telegram/ext/_choseninlineresulthandler.py

```diff
@@ -47,7 +47,8 @@ class ChosenInlineResultHandler(Handler[Update, CCT]):
             :class:`telegram.ext.ConversationHandler`.
         run_async (:obj:`bool`): Determines whether the callback will run asynchronously.
             Defaults to :obj:`False`.
-        pattern (:obj:`str` | `Pattern`, optional): Regex pattern. If not :obj:`None`, ``re.match``
+        pattern (:obj:`str` | `Pattern`, optional): Regex pattern. If not :obj:`None`,
```

Comment:
```suggestion
        pattern (:obj:`str` | :attr:`Pattern.pattern`, optional): Regex pattern. If not :obj:`None`,
```
do this? from https://docs.python.org/3/library/re.html#re.Pattern.pattern

Reply:
I used `:func:`re.Pattern <re.compile>`` in the other places - this prints `re.Pattern` but points to `re.compile`. this made sense for me b/c that's how you create pattern objects …. Added it here as well